airflow/operators/python.py

```diff
@@ -327,7 +327,13 @@ class PythonVirtualenvOperator(PythonOperator):
         processing templated fields, for examples ``['.sql', '.hql']``
     """
 
-    template_fields: Sequence[str] = ('requirements',)
+    template_fields: Sequence[str] = tuple(
+        {
+            'requirements',
+        }
+        | set(PythonOperator.template_fields)
+    )
```

Comment:
```suggestion
    template_fields: Sequence[str] = tuple({"requirements"} | set(PythonOperator.template_fields))
```
I think the comma in the first set is making black format the code strangely.